### PR TITLE
remove sessionId parameter from api calls

### PR DESCRIPTION
**Pull request by crobibero**
https://github.com/jellyfin/jellyfin/pull/3378

I didn't go up the chain to remove unused parameters as I don't know JS very well.

### CODE REVIEW

File: src/components/syncPlay/groupSelectionMenu.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ function showNewJoinGroupSelection (button, user, apiClient) {
         console.debug('No item is currently playing.');
     }
 
-    apiClient.sendSyncPlayCommand(sessionId, 'ListGroups').then(function (response) {
+    apiClient.sendSyncPlayCommand('ListGroups').then(function (response) {
```
Comment (JustAMan) [inline src/components/syncPlay/groupSelectionMenu.js:40]: @MrTimscampi doesn't this need an update to apiclient?..

Comment (MrTimscampi) [reply src/components/syncPlay/groupSelectionMenu.js:40]: It's done already, the PR there (and this one) are waiting for the backend PR.

Comment (JustAMan) [reply src/components/syncPlay/groupSelectionMenu.js:40]: This PR should update the version of apiclient used.

Comment (MrTimscampi) [reply src/components/syncPlay/groupSelectionMenu.js:40]: We need to release that first, but that needs the backend PR to be merged first.

Comment (joshuaboniface) [reply src/components/syncPlay/groupSelectionMenu.js:40]: I believe this was addressed below.

**Pull request merged**